pkg/util/membercluster_client.go

```diff
@@ -121,12 +121,13 @@ func buildClusterConfig(clusterName string, client client.Client) (*rest.Config,
 		return nil, fmt.Errorf("the api endpoint of cluster %s is empty", clusterName)
 	}
 
-	secretNamespace := cluster.Spec.SecretRef.Namespace
-	secretName := cluster.Spec.SecretRef.Name
-	if secretName == "" {
+	if cluster.Spec.SecretRef == nil || cluster.Spec.SecretRef.Name == "" {
```

Comment:
Do you think we should check the `Namespace` here?

Reply:
Actually, `Name` and `Namespace` fields are required in `SecretRef` schema. So, maybe we don't need check `Name` nor `Namespace` here?

Reply:
Oh, you are right.